cmd/kube-apiserver/app/server.go

```diff
@@ -181,6 +182,29 @@ func (s *APIServer) verifyPortalFlags() {
 	}
 }
 
+// dial attempts to open a TCP connection to the provided address, explicitly
+// enabling keep-alives with a one-second interval.
+//
+// TODO: This is copy-pasted from etcd.Client.dial(), since that method isn't public.
```

Comment:
Sorry for my ignorance here - but what should LICENCE file contain?
Any of files under Godeps/_workspace/src/github.com/coreos/go-etcd/etcd doesn't contain any licence-related header.